lms/djangoapps/course_api/views.py

```diff
@@ -270,3 +271,7 @@ def get_queryset(self):
             course for course in db_courses
             if unicode(course.id) in search_courses_ids
         ]
+
+
+def redirect_courses(request):
+    return HttpResponseRedirect("https://openlearninglibrary.mit.edu/courses/course-v1:OCW+Pre-7.01+1T2020/about")
```

Comment:
Kindly, avoid hard code URL.
You can create it on runtime
User `reverse` or 
something like

```
'{}/courses/{}/about'.format(base_url,request.course_id)
```

Reply:
how can we get `request.course_id`, we need to get the URL hardcoded although base can get from a reverse, I didn't do that because I had to make course locally, but I can do that.

Reply:
nit:
@mumerfarooq000 
can we create a top-level variable naming something similar to `BIOLOGY_COURSE_ID` and assign it a hardcoded value and then use it?

Reply:
@mumerfarooq000 
The example was just for the reference 😄 I didn't mean that you will exactly do this.